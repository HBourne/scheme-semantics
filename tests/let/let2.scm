[let [[x 5]]
    [let [[x 2]
          [y x]]
      [list y x]]]
eof