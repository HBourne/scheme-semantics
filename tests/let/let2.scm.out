V ==K { 5 , { 2 , { } } }
